src/dotnet/commands/dotnet-restore3/Program.cs

```diff
@@ -60,6 +60,12 @@ public static int Run(string[] args)
                     "Treat package source failures as warnings.",
                     CommandOptionType.NoValue);
 
+            // Use a boolean argument instead of a switch to match nuget.exe
```

Comment:
We don't have any tests for restore3 itself? I just realized that all the other _3 verbs have tests.